opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/activity/BaseAncHomeVisitActivity.java

```diff
@@ -236,9 +243,9 @@ public void redrawVisitUI() {
             }
         }
 
-        int res_color = valid ? R.color.white : R.color.light_grey;
-        tvSubmit.setTextColor(getResources().getColor(res_color));
-        tvSubmit.setOnClickListener(valid ? this : null); // update listener to null
+        isHomeVisitCompleted = valid;
```

Comment:
Do we need 2 different boolean variables or would one e.g. `isHomeVisitCompleted` just suffice considering they're updated at the same time/place?

Reply:
Good capture, actually I think I will move isHomeVisitCompleted out of that function because there is a condition for isValid that is not applicable to isHomeVisitCompleted. Let me modify!